src/pudl/output/ferc1.py

```diff
@@ -1159,9 +1174,10 @@ def _aggregatable_dimension_tags(
 
 def exploded_table_asset_factory(
     root_table: str,
-    table_names_to_explode: list[str],
+    table_names: list[str],
```

Comment:
i changed this so it's easier to copy/paste `explosion_args` in `create_exploded_table_assets` into a notebook to do:
```
args_explode = explosion_args[1]
args_generic = {
    "metadata_xbrl_ferc1":meta_w_dims,
    "calculation_components_xbrl_ferc1":calc_components,
    "tags":tags_df
}

exploder = Exploder(
    **args_explode|args_generic
)
```